pkg/datastore/example_multiinstance_test.go

```diff
@@ -22,6 +23,8 @@ func (p Person) String() string {
 	return fmt.Sprintf("%s: %d", p.Name, p.Age)
 }
 
+// Example for the multi-instance feature by creating records for different instances
+// and checks that correct instance can access data belonging to it.
```

Comment:
I propose changing this comment to this:

> Example for the multi-instance feature.
> Creates records for different instances and checks that the correct instance can access data belonging to it.

Reply:
@krishnamiriyala, I propose you fix this in this PR because right now the sentence (Example for the multi-instance feature by creating records for different instances and checks that correct instance can access data belonging to it.) is not grammatically correct